src/queries/useConfigurations.js

```diff
@@ -0,0 +1,48 @@
+import { useQuery } from 'react-query';
+
+import queryLimit from '../queryLimit';
+import useOkapiKy from '../useOkapiKy';
+import { useStripes } from '../StripesContext';
+
+
+export const configurationsApi = (module, configName, code) => {
+  const params = [];
+  if (module) {
+    params.push(`module=="${module}"`);
+  }
+  if (configName) {
+    params.push(`configName=="${configName}"`);
+  }
+  if (code) {
+    params.push(`code=="${code}"`);
+  }
+
+  const query = params.length ? `query=(${params.join(' and ')})` : '';
+
+  return `configurations/entries?${query}&limit=${queryLimit()}`;
+};
+
+const useConfigurations = ({ module, configName, code }) => {
+  const stripes = useStripes();
+  const ky = useOkapiKy();
+  const namespace = '@folio/stripes-core:QUERY_KEY_CONFIGURATIONS';
+
+  const { isFetching, data, error } = useQuery(
+    [namespace, configName, code],
+    async () => {
```

Comment:
@zburke I don't think we need `async` here since there is no `await` right?

Reply:
Or perhaps the `await` is missing before `ky.get`?

Reply:
I think you're correct that `async` here is superfluous. It's `useQuery` that would be calling `await` here, but given `ky.get()` is already async there's no value in restating it. I'll remove it. 

Thanks for mentioning this and helping to keep the code as clean as possible!